tests/optional_kernel_features/kernel_features_common.h

```diff
@@ -954,27 +964,33 @@ class kernel_submission_call;
         single_task_action, parallel_for_action, parallel_for_wg_action);   \
   }
 
-#define RUN_SUBMISSION_CALL_ND_RANGE(IS_EXCEPTION_EXPECTED, ERRC, QUEUE,      \
-                                     ATTRIBUTE, KERNEL_NAME, ...)             \
+#define RUN_SUBMISSION_CALL_ND_RANGE(SIZE, D, IS_EXCEPTION_EXPECTED, ERRC,    \
```

Comment:
Just to avoid wondering next time I read this :-)
```suggestion
/// Use a macro because we need to inject C++11 attributes
#define RUN_SUBMISSION_CALL_ND_RANGE(SIZE, D, IS_EXCEPTION_EXPECTED, ERRC,    \
```

Reply:
A good rationale to remove attributes from SYCL Next, by the way.